sdk/tables/azure-data-tables/CMakeLists.txt

```diff
@@ -70,10 +70,11 @@ set(
     src/private/policies/tenant_bearer_token_policy.hpp
     src/private/policies/timeout_policy.hpp
     src/private/serializers.hpp
+    src/private/table_constants.hpp
     src/private/url_encode.hpp
     src/private/xml_wrapper.hpp
     src/serializers.cpp
-    src/tables_clients.cpp
+    src/table_clients.cpp
```

Comment:
So, we split the header into two, why not also split the cpp into two, assuming it is a simple move and there's not much code to reuse, no shared helper functionality inside an unnamed namespace in the cpp file.
It must be easier to maintain (although I haven't seen the file).

Reply:
Since that is largely implementation detail, I'd leave that to George's tastes/preference to revisit as the library owner and avoid churn in the source cpp files that isn't critical.